<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CameraControlsDialog</class>
 <widget class="QDialog" name="CameraControlsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>288</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Camera Settings &amp; Controls</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="camSettings">
     <property name="title">
      <string>Camera Settings</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="nearPlane">
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="minimum">
         <double>0.000100000000000</double>
        </property>
        <property name="maximum">
         <double>10.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.001000000000000</double>
        </property>
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="farPlane">
        <property name="minimum">
         <double>20.000000000000000</double>
        </property>
        <property name="maximum">
         <double>100000000.000000000000000</double>
        </property>
        <property name="value">
         <double>100000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Near plane:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Far plane:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="navControls">
     <property name="title">
      <string>Flycam Controls</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <item>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="3" column="0">
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Left:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QLineEdit" name="backwardDisplay_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="3">
         <widget class="QLineEdit" name="rightDisplay_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="backwardDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Vertically Down:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QToolButton" name="backwardSet_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="rightDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="3">
         <widget class="QLineEdit" name="leftDisplay_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="6" column="2">
         <widget class="QToolButton" name="downSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Backward:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Primary</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="forwardDisplay_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="leftDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="2">
         <widget class="QToolButton" name="rightSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QLineEdit" name="downDisplay_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Secondary</string>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QToolButton" name="leftSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="5" column="4">
         <widget class="QToolButton" name="upSet_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="5" column="3">
         <widget class="QLineEdit" name="upDisplay_2">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="3" column="4">
         <widget class="QToolButton" name="leftSet_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QToolButton" name="upSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QToolButton" name="forwardSet_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="6" column="4">
         <widget class="QToolButton" name="downSet_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="downDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Right:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="forwardDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QToolButton" name="forwardSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Forward:</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Vertically Up:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="backwardSet">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="4" column="4">
         <widget class="QToolButton" name="rightSet_2">
          <property name="text">
           <string>Set</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QLineEdit" name="upDisplay">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="7" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Faster Movement:</string>
          </property>
         </widget>
        </item>
        <item row="7" column="1">
         <widget class="QComboBox" name="speedMod">
          <item>
           <property name="text">
            <string>Shift</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Alt</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Control</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Disabled</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="3" colspan="2">
         <widget class="QPushButton" name="resetAll">
          <property name="text">
           <string>Reset All to Default</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>nearPlane</tabstop>
  <tabstop>farPlane</tabstop>
  <tabstop>forwardSet</tabstop>
  <tabstop>backwardSet</tabstop>
  <tabstop>leftSet</tabstop>
  <tabstop>rightSet</tabstop>
  <tabstop>upSet</tabstop>
  <tabstop>downSet</tabstop>
  <tabstop>forwardSet_2</tabstop>
  <tabstop>backwardSet_2</tabstop>
  <tabstop>leftSet_2</tabstop>
  <tabstop>rightSet_2</tabstop>
  <tabstop>upSet_2</tabstop>
  <tabstop>downSet_2</tabstop>
  <tabstop>speedMod</tabstop>
  <tabstop>resetAll</tabstop>
  <tabstop>forwardDisplay</tabstop>
  <tabstop>backwardDisplay</tabstop>
  <tabstop>leftDisplay</tabstop>
  <tabstop>rightDisplay</tabstop>
  <tabstop>upDisplay</tabstop>
  <tabstop>downDisplay</tabstop>
  <tabstop>forwardDisplay_2</tabstop>
  <tabstop>backwardDisplay_2</tabstop>
  <tabstop>leftDisplay_2</tabstop>
  <tabstop>rightDisplay_2</tabstop>
  <tabstop>upDisplay_2</tabstop>
  <tabstop>downDisplay_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
